docs/developers/authorization.md

```diff
@@ -0,0 +1,107 @@
+# Authorization Set up
+
+  Authorization is the process of giving someone permission to do or have something. In multi-user systems, a system administrator defines for the system which users are allowed access to the system and what privileges of use.
+
+We have a standard access control matrix with 3 axes:
+
+- Actions: CREATE, READ, WRITE, DELETE, PURGE
+- Entities (object): User, Publisher, Package, Package Resource, …
+- Users: a user or type of user
+
```

Comment:
Might be worth defining a "permission" as a tuple of these and giving an example.

Also maybe copying over the access control table from gdocs would be good.